src/routes/EResourceViewRoute.js

```diff
@@ -39,18 +40,19 @@ class EResourceViewRoute extends React.Component {
       path: 'erm/packages/:{id}/content/%{packageContentsFilter}',
       records: 'results',
       limitParam: 'perPage',
-      perRequest: RECORDS_PER_REQUEST,
-      recordsRequired: '%{packageContentsCount}',
+      perRequest: resultCount.RESULT_COUNT_INCREMENT,
+      resultOffset: '%{resultOffset}',
       params: {
         filters: 'pkg.id==:{id}',
         sort: 'pti.titleInstance.name;asc',
         stats: 'true',
       },
     },
     query: {},
-    entitlementsCount: { initialValue: RECORDS_PER_REQUEST },
+    entitlementsCount: { initialValue: resultCount.INITIAL_RESULT_COUNT },
     packageContentsFilter: { initialValue: 'current' },
-    packageContentsCount: { initialValue: RECORDS_PER_REQUEST },
+    resultOffset: { initialValue: 0 },
```

Comment:
Can we be a bit more specific about what `result` we mean in `resultCount` and `resultOffset`?  We've got a pair of these resources that need these kinds of variables right now (entitlements is the other one) and could have more in the future so it can be confusing.
```suggestion
    packageContentsOffset: { initialValue: 0 },
```

Reply:
Yup, will change it